src/Graby.php

```diff
@@ -815,7 +815,12 @@ private function makeAbsoluteAttr($base, \DOMNode $e, $attr)
         $url = trim(str_replace('%20', ' ', $e->getAttribute($attr)));
         $url = str_replace(' ', '%20', $url);
 
-        $absolute = $this->makeAbsoluteStr($base, $url);
+        try {
+            $absolute = $this->makeAbsoluteStr($base, $url);
+        } catch (\Exception $exception) {
+            $this->logger->info('Wrong content url', ['url' => $url]);
+            $absolute = $url;
```

Comment:
If we failed to make it absolute, shouldn't we skip it instead?

```suggestion
            $absolute = false;
```

Reply:
I think we should keep this URL that the user can see the issue. Plus, we should keep the content, even if it's bad content. :/